### PR TITLE
Implemented Valid Parentheses

### PR DESCRIPTION
**Pull request by odiljon**

### CODE REVIEW

File: src/main/java/com/osattarov/leetcode/easy/ValidParentheses.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     * Validates parentheses.
+     *
+     * @param parentheses given parentheses
+     * @return true if given parentheses are valid, otherwise false
+     */
+    boolean isValid(String parentheses) {
+        int count = 0;
+
+        while (parentheses.length() >= count) {
+            parentheses = parentheses.replace("()", EMPTY);
```
Comment (odiljon) [inline src/main/java/com/osattarov/leetcode/easy/ValidParentheses.java]: @mavharsha As you know there are so many ways to do it but I chose easy one 😛

Comment (mavharsha) [reply src/main/java/com/osattarov/leetcode/easy/ValidParentheses.java]: Great way to solve the problem. But my only concern is time complexity. Wouldn't it be `O(n^2)` ?

Comment (odiljon) [reply src/main/java/com/osattarov/leetcode/easy/ValidParentheses.java]: I think most of the java internal methods take less than `n^2` but I might be wrong. Let me try with different approach using some data structure.

Comment (mavharsha) [reply src/main/java/com/osattarov/leetcode/easy/ValidParentheses.java]: 👍 👏 🌮 🥇

**Pull request merged**